pyintelowl/pyintelowl.py

```diff
@@ -634,6 +634,42 @@ def _get_observable_classification(self, value: str) -> str:
 
         return classification
 
+    def download_sample(self, job_id: int) -> dict:
```

Comment:
```
def download_sample(self, job_id: int) -> bytes:
```

We should return directly the `response.content` which is basically the file buffer in bytes format.

Reply:
Remember to make corresponding changes in the CLI command too.